airgun/entities/template.py

```diff
@@ -28,16 +30,22 @@ def clone(self, entity_name, values):
         view = self.navigate_to(self, 'Clone', entity_name=entity_name)
         view.fill(values)
         view.submit.click()
+        view.flash.assert_no_error()
+        view.flash.dismiss()
 
     def lock(self, entity_name):
         view = self.navigate_to(self, 'All')
         view.search(entity_name)
         view.table.row(name=entity_name)['Actions'].widget.fill('Lock')
+        view.flash.assert_no_error()
+        view.flash.dismiss()
 
     def unlock(self, entity_name):
         view = self.navigate_to(self, 'All')
         view.search(entity_name)
         view.table.row(name=entity_name)['Actions'].widget.fill('Unlock')
+        view.flash.assert_no_error()
```

Comment:
if there's browser alert - it should be handled before any other interaction with the page (like dismissing notifications). Please move `self.browser.handle_alert()` before these lines